### PR TITLE
[vi] Logic models

### PR DESCRIPTION
**Pull request by tuanhm2911**
Finished translation Vietnamese in topic Logic models.
Welcome to review.

### CODE REVIEW

**Comment by shervinea** [comment]
Thank you for your dedication @tuanhm2911!

File: vi/cs-221-logic-models.md [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+**65. By X and Y**
+
+&#10230; Bởi X và Y
+
+<br>
+
+
+**66. The Artificial Intelligence cheatsheets are now available in [target language].**
+
+&#10230; Trí tuệ nhân tạo cheatsheats hiện đã có vơi ngôn ngữ [Tiếng Việt]
```
Comment (damminhtien) [inline vi/cs-221-logic-models.md]:
```suggestion
&#10230; Trí tuệ nhân tạo cheatsheats hiện đã có với ngôn ngữ [Tiếng Việt]
```

File: vi/cs-221-logic-models.md [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+**17. Remark: M(KB) denotes the set of models compatible with all the constraints of the knowledge base.**
+
+&#10230; Ghi chú: M(KB) biểu thị tập hợp các mô hình tương thích với tất cả các ràng buộc của cơ sở tri thức.
+
+<br>
+
+
+**18. Relation between formulas and knowledge base - We define the following properties between the knowledge base KB and a new formula f:**
+
+&#10230; Mối liên hệ giữa công thức và cơ sở tri thức - Chúng tôi xác định các thuộc tính sau giữa KB cơ sở tri thức và công thức mới f:
```
Comment (damminhtien) [inline vi/cs-221-logic-models.md]:
```suggestion
&#10230; Mối liên hệ giữa công thức và cơ sở tri thức - Chúng tôi định nghĩa các thuộc tính sau giữa KB cơ sở tri thức và công thức mới f:
```

**Comment by shervinea** [comment]
Hi @tuanhm2911, would the suggestions of @damminhtien sound good to you? Please feel to give your feedback when you get the time. After that, we'll be able merge all your hard work!

File: vi/cs-221-logic-models.md [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+**57. [Basics, Notations, Model, Interpretation function, Set of models]**
+
+&#10230; [Khái niệm cơ bản, Ký hiệu, Mô hình, Hàm diễn giải, Bộ mô hình]
+
+<br>
+
+
+**58. [Knowledge base, Definition, Probabilistic interpretation, Satisfiability, Relationship with formulas, Forward inference, Rule properties]**
+
+&#10230; [Cơ sở tri thức, Định nghĩa, Giải thích xác suất, Hài lòng, Mối quan hệ với các công thức, Suy luận chuyển tiếp, Thuộc tính quy tắc]
```
[the remark this answers is not in the document]
Comment (tuanhm2911) [reply vi/cs-221-logic-models.md]: Thank you for your suggestion

File: vi/cs-221-logic-models.md [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+**12. Knowledge base**
+
+&#10230; Cơ sở tri thức
+
+<br>
+
+
+**13. Definition ― The knowledge base KB is the conjunction of all formulas that have been considered so far. The set of models of the knowledge base is the intersection of the set of models that satisfy each formula. In other words:**
+
+&#10230; Định nghĩa - Cơ sở tri thức KB là sự kết hợp của tất cả các công thức đã được xem xét cho đến nay. Tập hợp các mô hình của cơ sở tri thức là giao điểm của tập hợp các mô hình thỏa mãn từng công thức. Nói cách khác:
```
[the remark this answers is not in the document]
Comment (tuanhm2911) [reply vi/cs-221-logic-models.md]: Thank you for your suggestion

File: vi/cs-221-logic-models.md (hunk 2 above)
Comment (tuanhm2911) [reply vi/cs-221-logic-models.md]: Thank you for your suggestion

File: vi/cs-221-logic-models.md [hunk 5]
```diff
@@ -0,0 +1,462 @@
...
+**19. [Name, Mathematical formulation, Illustration, Notes]**
+
+&#10230; [Tên, Công thức toán học, Minh họa, Ghi chú]
+
+<br>
+
+
+**20. [KB entails f, KB contradicts f, f contingent to KB]**
+
+&#10230; [KB đòi hỏi f, KB mâu thuẫn với f, f phụ thuộc vào KB]
```
[the remark this answers is not in the document]
Comment (tuanhm2911) [reply vi/cs-221-logic-models.md]: Thank you for your suggestion

**Comment by shervinea** [comment]
Thank you @tuanhm2911 and @damminhtien for all your work! Moving forward with the merge now.

**Pull request merged**